src/game_stats.py

```diff
@@ -57,10 +59,34 @@ def log_end_game(self):
     def log_failed_run(self):
         self._runs_failed += 1
 
-    def get_current_game_length(self):
+    def pause_timer(self):
+        """ Pauses the timer """
```

Comment:
No need to add comments if they are the same as the function name. There is not going to be any quality team that forces us to have every function documented with comments :yum: